### PR TITLE
Draft: New Scenario Added

### PR DESCRIPTION
**Pull request by friedcucumber**

### CODE REVIEW

File: CS/PropertyDescription/Readme.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# DevExpress .NET MAUI HTML Edit - Edit HTML in DataGridView CRUD Views
+
+This scenario uses the [HTML Edit Control](https://docs.devexpress.com/MAUI/404635) to show and edit HTML markup of the selected DataGridView item.
```
Comment (Abadzhev) [inline CS/PropertyDescription/Readme.md]: Do we need to call it a "scenario"? I know you decided to call a bunch of examples "scenarios" in the documentation. But let's say I found this example in Google. In that case, this word makes little sense to me ("This scenario uses..."). We can use the word "project" or "example" - something simple. 

Another issue is the word "markup". If I see "markup", I think "source code" - tags, attributes, etc. So the way you wrote this sentence, I think that maybe you talk about a code/syntax editor, i.e. I would actually be editing HTML code with all the tags. 

```suggestion
This example uses a DataGridView to display a list of real estate properties for sale. Each item in the list has an associated description in HTML format. When you view or edit individual property details, the DevExpress [HTML Edit Control](https://docs.devexpress.com/MAUI/404635) displays the formatted description.
```

File: CS/PropertyDescription/Readme.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+<img src="images/property-description-overview@2x.png?v=23.2" width="50%"/>
```
Comment (Abadzhev) [inline CS/PropertyDescription/Readme.md]: Please fix currency formatting. "780000 $" must be "$780,000". 

Make sure that the app uses proper currency formatting for prices and re-take screenshots.

File: CS/PropertyDescription/Readme.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+## Implementation Details
+
+* In this scenario, the HTML Edit control is displayed in the [DataGridView](https://docs.devexpress.com/MAUI/403255/data-grid/data-grid)'s Detail and Edit [CRUD forms](https://docs.devexpress.com/MAUI/404456/data-grid/crud/customize-detail-forms). You can use the [DetailFormTemplate](https://docs.devexpress.com/MAUI/DevExpress.Maui.DataGrid.DataGridView.DetailFormTemplate) and [DetailEditFormTemplate](https://docs.devexpress.com/MAUI/DevExpress.Maui.DataGrid.DataGridView.DetailEditFormTemplate) properties to define the page that is used to view and edit the selected DataGridView's item.
```
Comment (Abadzhev) [inline CS/PropertyDescription/Readme.md]: Once again, I'm avoiding the word "scenario". 

Also removing a couple of passive voice constructs: "is displayed", "that is used". 

And if you list properties, you don't have to use the article (per our guidelines). 

As for possessive at the end. To me, "the selected grid's item" is the same as "the item of the selected grid". But "the selected grid item" is "the selected item of the grid". 

```suggestion
* The HTML Edit control is part of [DataGridView](https://docs.devexpress.com/MAUI/403255/data-grid/data-grid)'s Detail and Edit [CRUD forms](https://docs.devexpress.com/MAUI/404456/data-grid/crud/customize-detail-forms). You can use [DetailFormTemplate](https://docs.devexpress.com/MAUI/DevExpress.Maui.DataGrid.DataGridView.DetailFormTemplate) and [DetailEditFormTemplate](https://docs.devexpress.com/MAUI/DevExpress.Maui.DataGrid.DataGridView.DetailEditFormTemplate) properties to define pages that display and edit the selected DataGridView item.
```

File: CS/PropertyDescription/Readme.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+* Set the [HtmlEdit.IsReadOnly](https://docs.devexpress.com/MAUI/404635/html-edit/html-edit#html-edit-availability) to true to disable editing.
```
Comment (Abadzhev) [inline CS/PropertyDescription/Readme.md]:
```suggestion
* Set the [HtmlEdit.IsReadOnly](https://docs.devexpress.com/MAUI/404635/html-edit/html-edit#html-edit-availability) property to `true` to disable editing.
```

File: CS/PropertyDescription/Readme.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+* Use the [HtmlEdit.GetHtmlAsync](https://docs.devexpress.com/MAUI/404637/html-edit/load-and-obtain-markup#retrieve-the-displayed-content) method to obtain the HTML Edit content. After that, you can save it to a property on the [OnNavigatingFrom](https://learn.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.page.onnavigatingfrom?view=winrt-22621) event.
```
Comment (Abadzhev) [inline CS/PropertyDescription/Readme.md]:
```suggestion
* Use the [HtmlEdit.GetHtmlAsync](https://docs.devexpress.com/MAUI/404637/html-edit/load-and-obtain-markup#retrieve-the-displayed-content) method to obtain the current HTML markup from the editor. You can save it to the associated property on the [OnNavigatingFrom](https://learn.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.page.onnavigatingfrom?view=winrt-22621) event.
```

File: CS/PropertyDescription/MauiProgram.cs [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+
+namespace PropertyDescriptionHTMLEdit;
+
+public static class MauiProgram {
+    public static MauiApp CreateMauiApp() {
+        var builder = MauiApp.CreateBuilder();
+        builder
+            .UseMauiApp<App>()
+            .UseDevExpress(useLocalization: true)            
+            .ConfigureFonts(fonts => {
```
Comment (JaneySprings) [inline CS/PropertyDescription/MauiProgram.cs:16]: You can remove `.UseCompatibility()` and unused fonts

File: CS/PropertyDescription/MauiProgram.cs [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+                fonts.AddFont("OpenSans-Regular.ttf", "OpenSansRegular");
+                fonts.AddFont("univia-pro-regular.ttf", "Univia-Pro");
```
Comment (friedcucumber) [inline CS/PropertyDescription/MauiProgram.cs]:
```suggestion

```

File: CS/PropertyDescription/MauiProgram.cs [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+                fonts.AddFont("roboto-bold.ttf", "Roboto-Bold");
+                fonts.AddFont("roboto-regular.ttf", "Roboto");
+            })
+            .UseMauiCompatibility();
```
Comment (friedcucumber) [inline CS/PropertyDescription/MauiProgram.cs]:
```suggestion

```

File: CS/PropertyDescription/MauiProgram.cs [hunk 9]
```diff
@@ -0,0 +1,26 @@
...
+        var builder = MauiApp.CreateBuilder();
+        builder
+            .UseMauiApp<App>()
+            .UseDevExpress(useLocalization: true)            
+            .ConfigureFonts(fonts => {
+                fonts.AddFont("OpenSans-Regular.ttf", "OpenSansRegular");
+                fonts.AddFont("univia-pro-regular.ttf", "Univia-Pro");
+                fonts.AddFont("roboto-bold.ttf", "Roboto-Bold");
+                fonts.AddFont("roboto-regular.ttf", "Roboto");
+            })
```
Comment (friedcucumber) [inline CS/PropertyDescription/MauiProgram.cs]:
```suggestion
            });

```

File: CS/PropertyDescription/Models/PropertyModel.cs [hunk 10]
```diff
@@ -0,0 +1,38 @@
...
+    // private ObservableCollection<PropertyItem> items;
+
+    [JsonPropertyName("ID")] public int Id { get; set; }
+    [JsonPropertyName("Address")] public string Address { get; set; }
+    [JsonPropertyName("Beds")] public byte Beds { get; set; }
+    [JsonPropertyName("Baths")] public byte Baths { get; set; }
+    [JsonPropertyName("HouseSize")] public int HouseSize { get; set; }
+    [JsonPropertyName("LotSize")] public double LotSize { get; set; }
+    [JsonPropertyName("Price")] public int Price { get; set; }
+    // [JsonPropertyName("Features")] public string Features { get; set; }
```
Comment (friedcucumber) [inline CS/PropertyDescription/Models/PropertyModel.cs]:
```suggestion
```

File: CS/PropertyDescription/Models/PropertyModel.cs [hunk 11]
```diff
@@ -0,0 +1,38 @@
+using System.Collections.ObjectModel;
+using System.Text.Json.Serialization;
+using DevExpress.Maui.Core;
+
+namespace PropertyDescriptionHTMLEdit.Models;
+
+
+public class Property : BindableBase
+{
+    // private ObservableCollection<PropertyItem> items;
```
Comment (friedcucumber) [inline CS/PropertyDescription/Models/PropertyModel.cs]:
```suggestion
```

File: CS/PropertyDescription/ViewModels/PropertyDetailFormViewModel.cs [hunk 12]
```diff
@@ -0,0 +1,35 @@
+using System.Collections.ObjectModel;
+using DevExpress.Maui.Core;
+// using PropertyDescriptionHTMLEdit.DataLayer;
+using PropertyDescriptionHTMLEdit.Models;
+
+namespace PropertyDescriptionHTMLEdit.ViewModels;
+
+public class PropertyDetailFormViewModel : DetailFormViewModel
+{
+    // private PropertyViewModel customersViewModel => (PropertyViewModel)DataControlContext;
```
Comment (friedcucumber) [inline CS/PropertyDescription/ViewModels/PropertyDetailFormViewModel.cs]:
```suggestion
```

File: CS/PropertyDescription/ViewModels/PropertyDetailFormViewModel.cs [hunk 13]
```diff
@@ -0,0 +1,35 @@
...
+    // private CrmContext crmContext => customersViewModel.CrmContext;
```
Comment (friedcucumber) [inline CS/PropertyDescription/ViewModels/PropertyDetailFormViewModel.cs]:
```suggestion
```

File: CS/PropertyDescription/ViewModels/PropertyDetailFormViewModel.cs [hunk 14]
```diff
@@ -0,0 +1,35 @@
...
+
+    // public Command<bool?> EditCommand { get; }
```
Comment (friedcucumber) [inline CS/PropertyDescription/ViewModels/PropertyDetailFormViewModel.cs]:
```suggestion
```

File: CS/PropertyDescription/ViewModels/PropertyDetailFormViewModel.cs [hunk 15]
```diff
@@ -0,0 +1,35 @@
...
+    public Property Property
+    {
+        get => ((Property)Item);
+        set
+        {
+            // ((Property)Item).Employee = value;
```
Comment (friedcucumber) [inline CS/PropertyDescription/ViewModels/PropertyDetailFormViewModel.cs]:
```suggestion
```

File: CS/PropertyDescription/ViewModels/PropertyDetailFormViewModel.cs [hunk 16]
```diff
@@ -0,0 +1,35 @@
...
+            // crmContext.SaveChanges();
```
Comment (friedcucumber) [inline CS/PropertyDescription/ViewModels/PropertyDetailFormViewModel.cs]:
```suggestion
```

File: CS/PropertyDescription/ViewModels/PropertyViewModel.cs [hunk 17]
```diff
@@ -0,0 +1,74 @@
...
+    {
+        return Task.Run(() => LoadDataFromJson());
+    }
+
+    private async void LoadDataFromJson()
+    {
+        IsDataLoading = true;
+
+        var json = await FileSystem.OpenAppPackageFileAsync("Homes.json");
+        // StreamReader reader = new StreamReader(json);
```
Comment (friedcucumber) [inline CS/PropertyDescription/ViewModels/PropertyViewModel.cs]:
```suggestion
```

File: CS/PropertyDescription/ViewModels/PropertyViewModel.cs [hunk 18]
```diff
@@ -0,0 +1,74 @@
...
+        // string line = "";
```
Comment (friedcucumber) [inline CS/PropertyDescription/ViewModels/PropertyViewModel.cs]:
```suggestion
```

File: CS/PropertyDescription/ViewModels/PropertyViewModel.cs [hunk 19]
```diff
@@ -0,0 +1,74 @@
...
+        // while ((line = reader.ReadLine()) != null)
```
Comment (friedcucumber) [inline CS/PropertyDescription/ViewModels/PropertyViewModel.cs]:
```suggestion
```

File: CS/PropertyDescription/ViewModels/PropertyViewModel.cs [hunk 20]
```diff
@@ -0,0 +1,74 @@
...
+        // {
```
Comment (friedcucumber) [inline CS/PropertyDescription/ViewModels/PropertyViewModel.cs]:
```suggestion
```

File: CS/PropertyDescription/ViewModels/PropertyViewModel.cs [hunk 21]
```diff
@@ -0,0 +1,74 @@
...
+        //     Console.WriteLine(line);
+        // }
```
Comment (friedcucumber) [inline CS/PropertyDescription/ViewModels/PropertyViewModel.cs]:
```suggestion
```

File: CS/PropertyDescription/ViewModels/PropertyViewModel.cs [hunk 22]
```diff
@@ -0,0 +1,74 @@
...
+    private async void LoadDataFromJson()
+    {
+        IsDataLoading = true;
+
+        var json = await FileSystem.OpenAppPackageFileAsync("Homes.json");
+        // StreamReader reader = new StreamReader(json);
+        // string line = "";
+        // while ((line = reader.ReadLine()) != null)
+        // {
+        //     Console.WriteLine(line);
```
Comment (friedcucumber) [inline CS/PropertyDescription/ViewModels/PropertyViewModel.cs]:
```suggestion
```

File: CS/PropertyDescription/ViewModels/PropertyDetailFormViewModel.cs [hunk 23]
```diff
@@ -0,0 +1,30 @@
+using System.Collections.ObjectModel;
+using DevExpress.Maui.Core;
+// using PropertyDescriptionHTMLEdit.DataLayer;
```
Comment (friedcucumber) [inline CS/PropertyDescription/ViewModels/PropertyDetailFormViewModel.cs]:
```suggestion
```

**Pull request merged**